pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: browser_launcher
 description: Provides a standardized way to launch web browsers for testing and tools.
 
-version: 0.1.6
+version: 0.1.7
 
 author: Dart Team <misc@dartlang.org>
```

Comment:
@devoncarew – while you're here, remove this. Will need to be dropped before publishing w/ recent SDKs.

Reply:
done!